src/store/metadata/stub/content/editions/pupils-at-expected-standards.json

```diff
@@ -0,0 +1,130 @@
+  "editions": [
+    {
+      "@id": "https://staging.idpd.uk/datasets/pupils-at-expected-standards/editions/2023-03",
+      "@type": "dcat:Dataset",
+      "in_series": "https://staging.idpd.uk/datasets/pupils-at-expected-standards",
+      "identifier": "2023-03",
+      "title": "Pupils at expected standards at the end of key stage 2",
+      "summary": "Percentage of pupils in state-funded schools meeting the expected standard in reading, writing and maths at the end of key stage 2, England (state-funded schools), academic years 2015/16 - 2021/22",
+      "description": "This dataset shows the percentage of pupils in state-funded schools meeting the expected standard in reading, writing and maths at the end of key stage 2, in England state-funded schools, for academic years 2015/16 - 2021/22.",
+      "publisher": "https://staging.idpd.uk/publishers/office-for-national-statistics",
+      "creator": "ofcom",
+      "contact_point": {
+        "name": "Ofcom Open Data",
+        "email": "mailto:open.data@ofcom.org.uk"
+      },
+      "topics": [
+        "https://staging.idpd.uk/topics/children-education-and-skills"
+      ],
+      "frequency": "monthly",
+      "keywords": [
+        "reading",
+        "writing",
+        "maths",
+        "subnational",
+        "region",
+        "county",
+        "state-funded-school",
+        "key-stage-2-attainment"
+      ],
+      "licence": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+      "issued": "2023-03-31T12:15:00Z",
+      "modified": "2023-08-23T15:55:00Z",
+      "spatial_resolution": [
+        "N09",
+        "S12",
+        "W06",
+        "E92",
+        "K02"
+      ],
+      "spatial_coverage": "K02000001",
+      "temporal_resolution": [
+        "P1M",
+        "P3M"
+      ],
+      "temporal_coverage": {
+        "start": "2022-09-01T00:00:00Z",
+        "end": "2022-09-30T23:59:00Z"
+      },
+      "versions_url": "https://staging.idpd.uk/datasets/pupils-at-expected-standards/editions/2023-03/versions",
+      "versions": [
+        {
+          "@id": "https://staging.idpd.uk/datasets/pupils-at-expected-standards/editions/2023-03/versions/1",
+          "issued": "2023-03-31T12:15:00Z"
+        }
+      ],
+      "next_release": "2024-04-19T09:30:00Z",
+      "table_schema": {
+        "columns": [
+          {
+            "name": "period",
+            "datatype": "string",
+            "titles": "Period",
+            "description": "The time period associated with the observation."
+          },
+          {
+            "name": "areacd",
+            "datatype": "string",
+            "titles": "ONS Geography Code",
+            "description": "The geography code associated with the observation."
+          },
+          {
+            "name": "areanm",
+            "datatype": "string",
+            "titles": "Area Name",
+            "description": "The name assiciated with The geography code "
```

Comment:
Typo: assiciated -> associated.
Second 'The' shouldn't be capitalised.

Reply:
done